tests/RRuleTest.php

```diff
@@ -3191,6 +3191,18 @@ public function humanReadableStrings()
 				"daily",
 				"daily"
 			),
+			array(
+				"DTSTART=20170202T161500Z\nRRULE:FREQ=MONTHLY;BYMONTHDAY=2",
```

Comment:
Tests are still breaking, probably because `DTSTART` takes `:` instead of `=`

Reply:
Sorry. I think I got it straight now.